packages/cli/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@zowe/cli",
-  "version": "8.0.0-next.202407181904",
-  "zoweVersion": "V3",
+  "version": "8.0.0-next.202407112150",
```

Comment:
We don't want to change the `version` property in this PR, just `zoweVersion` 😋

Reply:
my tests dont change that. i think i just had the wrong original version or something